tests/bluetooth/host/buf/bt_buf_get_rx/testcase.yaml

```
@@ -0,0 +1,20 @@
+common:
+    tags: test_framework bluetooth host
+tests:
+  bluetooth.host.bt_buf_get_rx.default:
+     type: unit
+  bluetooth.host.bt_buf_get_rx.hci_acl_flow_control:
+     type: unit
+     # enable CONFIG_BT_HCI_ACL_FLOW_CONTROL
+     extra_configs:
+      - CONFIG_BT_CENTRAL=y
```

the test is called: `hci_acl_flow_control`

so in addition / instead of a comment, maybe make it explicit that this test is enabling this, although it's the default.
```suggestion
     extra_configs:
      - CONFIG_BT_CENTRAL=y
      - CONFIG_BT_HCI_ACL_FLOW_CONTROL=y
```

that will make it more clear to the reader what we are testing, and removes the need for a comment.